docs/requirements-docs.txt

```diff
@@ -1,4 +1,4 @@
-numpydoc==0.9.1
+numpydoc==0.8.0
```

Comment:
Could you please add a comment here about why we're pinning `numpydoc`. It'll hopefully help us later when deciding when we can update the version used